src/main/java/codes/biscuit/skyblockaddons/mixins/MixinEntityRenderer.java

```diff
@@ -43,6 +46,18 @@ private void removeEntities(List<Entity> list) {
                 double auctionZ = -78.5;
                 list.removeIf(listEntity -> listEntity.getDistance(auctionX, auctionY, auctionZ) <= 3 && (listEntity.posX != auctionX || listEntity.posY != auctionY || listEntity.posZ != auctionZ));
             }
+            if (!SkyblockAddons.getInstance().getConfigValues().isDisabled(Feature.DISABLE_PROFILE_VIEW_IF_DRAGON_ALIVE)) {
+                Minecraft mc = Minecraft.getMinecraft();
+                for (Entity elist : mc.theWorld.loadedEntityList) {
+                    if (elist instanceof EntityDragon) {
+                        double coreX = -670;
+                        double coreY = 9;
+                        double coreZ = -275;
+
+                        list.removeIf(listEntity -> listEntity.getDistance(coreX, coreY, coreZ) <= 70 && listEntity instanceof EntityOtherPlayerMP);
```

Comment:
Would checking if `listEntity instanceof EntityOtherPlayerMP` first be faster?

Reply:
Probably better, thank you.